src/launch/main.c

```diff
@@ -69,8 +69,8 @@ static bool             main_arg_audit = false;
 static const char *     main_arg_broker = BINDIR "/dbus-broker";
 static bool             main_arg_force = false;
 static const char *     main_arg_listen = NULL;
-static const char *     main_arg_scope = "system";
 static const char *     main_arg_policypath = NULL;
+static bool             main_arg_user_scope = false;
```

Comment:
Mh. I understand having a string is cumbersome, but a boolean reads weird as well. Mhh. Technically this should be an enum. Stupid C.

Reply:
Wow, how did this comment end up in this commit? *sigh*